tensorflow/compiler/xla/service/hlo_evaluator.cc

```diff
@@ -1031,9 +1031,10 @@ Status HloEvaluator::HandleCall(HloInstruction* call) {
   }
 
   HloEvaluator embedded_evaluator;
-  Literal result =
-      embedded_evaluator.Evaluate<const Literal*>(*computation, arg_literals)
-          .ConsumeValueOrDie();
+
+  Literal result;
```

Comment:
Minor: it looks weird, but you can put the declaration in the macro (this is common in our code);

TF_ASSIGN_OR_RETURN(Literal result, embedded_evaluator...

Also, there are other occurrences of ConsumeValueOrDie in this file in similar scenarios - perhaps we just fix all of them at once?

Reply:
+1 on putting the declaration in the macro.

It seems like the other offenders are HandleFusion and HandleConditional.

Reply:
Thanks, I've change the format now.
I originally only did HandleCall as that was the only one affecting us, but I've extended it to HandleFusion and HandleConditional.